levechad/settings.py

```diff
@@ -229,6 +231,13 @@
     },
 }
 
+
 # Geocoding settings
-LOCATOR = 'NOMINATIM'
+class LocatorTypes(Enum):
+    nominatim = 1
+    google = 2
+    arcgis = 3
+
+
+LOCATOR = LocatorTypes(LocatorTypes.nominatim)
```

Comment:
Replace with `LOCATOR = LocatorTypes.nominatim`, no need for the second `LocatorTypes()`